news/reversed.rst

```diff
@@ -0,0 +1,9 @@
+New features
+------------
+
+The ``__reversed__`` method has been added to the following iterables:
```

Comment:
```suggestion
The built-in ``reversed()`` function now works with the following iterables:
```

Release notes are meant for users, so the focus is on what you can do rather than the details of special methods.